cmake/ROS2.cmake

```diff
@@ -0,0 +1,65 @@
+cmake_minimum_required(VERSION 3.5)
+project(odom_transform)
+
+# Default to C99
+if(NOT CMAKE_C_STANDARD)
+  set(CMAKE_C_STANDARD 99)
+endif()
+
+# Default to C++14
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
```

Comment:
Can you enforce the cmake to use CMAKE_CXX_STANDARD 17?

Reply:
Ok will do that